src/layout/grid/gridColumn.jsx

```diff
@@ -332,27 +335,38 @@ const GridColumn = React.forwardRef(
             sm,
             style,
             xl,
+            responsiveQueryType,
             ...otherProps
         } = props;
 
         const classes = useStyles(props);
 
+        const hasContainerQueriesEnabled = responsiveQueryType === 'container';
+        console.log('hasContainerQueriesEnabled', hasContainerQueriesEnabled);
```

Comment:
1) Please remove `console.log()` when done debugging.
2) Please leave Line 345 blank (empty line between declaration on 344 and the multiline one on 346).